content/pages/index.html

```diff
@@ -155,62 +155,51 @@ <h3 id="heading">Test Run for Firefox RHEL6.5 (Desktop)</h3>
                 nextBtn = document.getElementById('next'),
                 prevBtn = document.getElementById('prev'),
                 shown = document.getElementsByClassName('shown')[0],
-                hidden =  document.getElementsByClassName('hidden-right')[0],
+                hidden =  document.getElementsByClassName('hidden')[0],
                 index = document.getElementById('imgIndex');
 
-                index.textContent = (imgIndex + 1) + '/' + imgsLength;
-
-                shown.style.backgroundImage = 'url(' + imgs[0].img + ')';
-                heading.textContent = imgs[0].title;
-                subtitle.textContent = imgs[0].subtitle;
-
-                nextBtn.addEventListener('mouseover', function(){
-                    hidden.className = 'img hidden-right';
-                });
+            index.textContent = (imgIndex + 1) + '/' + imgsLength;
 
-                prevBtn.addEventListener('mouseover', function(){
-                    hidden.className = 'img hidden-left';
-                });
+            shown.style.backgroundImage = 'url(' + imgs[0].img + ')';
+            heading.textContent = imgs[0].title;
+            subtitle.textContent = imgs[0].subtitle;
 
-                nextBtn.addEventListener('click', function(){
-                    imgIndex++;
+            function addClick(direction){
+                imgIndex += direction;
 
-                    if(imgIndex > imgsLength - 1){
-                        imgIndex = 0;
-                    }
+                if(imgIndex < 0){
+                    imgIndex = imgsLength - 1;
+                }
 
-                    index.textContent = (imgIndex + 1) + '/' + imgsLength;
+                if(imgIndex > imgsLength - 1){
+                    imgIndex = 0;
+                }
 
-                    shown.className = 'img hidden-right';
-                    hidden.className = 'img shown';
-
-                    hidden.style.backgroundImage = 'url(' + imgs[imgIndex].img + ')';
-                    heading.textContent = imgs[imgIndex].title;
-                    subtitle.textContent = imgs[imgIndex].subtitle;
+                index.textContent = (imgIndex + 1) + '/' + imgsLength;
 
-                    shown = document.getElementsByClassName('shown')[0];
-                    hidden =  document.getElementsByClassName('hidden-right')[0];
-                });
+                hidden.style.left = "0";
+                shown.className = 'img hidden';
+                hidden.className = 'img shown';
```

Comment:
Try using the element.classList.add and remove functions from:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
Also these 2 lines should be moved under the next 3 lines where you set background and text. The reason is making code more readable and organizing similar pieces together. If you move the lines below then you get:
1) change background & text
2) swap shown and hidden elements CSS styles
3) then swap the elements to which the `shown` and `hidden` variables point to
4) then call `changeDirection` to adjust positioning of the elements.